eclair-core/src/main/scala/fr/acinq/eclair/wire/ChannelCodecs.scala

```diff
@@ -156,28 +156,30 @@ object ChannelCodecs extends Logging {
       ("sentAfterLocalCommitIndex" | uint64overflow) ::
       ("reSignAsap" | bool8)).as[WaitingForRevocation]
 
-  val localCodec: Codec[Origin.Local] = (
-    ("id" | uuid) ::
-      ("sender" | provide(Option.empty[ActorRef]))
-    ).as[Origin.Local]
+  val localColdCodec: Codec[Origin.LocalCold] = (
+    ("id" | uuid)).as[Upstream.Local].as[Origin.LocalCold]
```

Comment:
nit: unnecessary parenthesis:
```suggestion
  val localColdCodec: Codec[Origin.LocalCold] = ("id" | uuid).as[Upstream.Local].as[Origin.LocalCold]
```

Reply:
Done with c46fff704.